shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -579,7 +694,14 @@ - (void)setSemanticsEnabled:(BOOL)enabled {
     return;
   }
   _semanticsEnabled = enabled;
-  [_viewController notifySemanticsEnabledChanged];
+
+  // Update all view controllers' bridges.
+  NSEnumerator* viewControllerEnumerator = [_viewControllers objectEnumerator];
+  FlutterViewController* nextViewController;
+  while ((nextViewController = [viewControllerEnumerator nextObject])) {
+    [nextViewController notifySemanticsEnabledChanged];
+  }
```

Comment:
Should this change also update the embedder API's `update_semantics_callback` callback?

Reply:
Great find! I've added a TODO to it.